website/config.toml

```diff
@@ -30,6 +30,11 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
     weight = -101
     pre = "<i class='fas fa-book pr-2'></i>"
     url = "./docs/"
+  [[menu.main]]
+    name = "Blog"
+    weight = -100
+    pre = "<i class='fas fa-blog pr-2'></i>"
+    url = "../main/blog/"
```

Comment:
@npepin-hub This is a great idea, only thing is we ideally never want user to go to main branch unless they choose to, can we point it to the latest instead?